mdv.js

```diff
@@ -34,45 +34,43 @@ const argv = require('yargs')
 let exitCode = 0;
 const options = argv;
 
-
 for (let a of argv._) {
-    glob(a, options, function (er, files) {
-        for (let file of files) {
-            const s = fs.readFileSync(a,'utf8');
-            options.source = file;
-            const result = validator.validate(s,options);
+    const files = glob.sync(a, { debug: !!options.debug });
```

Comment:
We were passing `options` before but the only `flag` that this CLI and `glob` have in common, so I simplified it.